### ISSUE TITLE
`werkzeug.urls` deprecation warnings with Werkzeug 2.3.0

### ISSUE THREAD

**Issue by The-Compiler** [csernazs/pytest-httpserver#242]
Werkzeug 2.3.0 [has deprecated](https://werkzeug.palletsprojects.com/en/2.3.x/changes/#version-2-3-0) most of the `werkzeug.urls` module:

- https://github.com/pallets/werkzeug/pull/2608

Due to that, running something like:

```python
import requests

def test_httpserver(httpserver):
    query = {"k": "v"}
    httpserver.expect_request("/", query_string=query).respond_with_data("OK")
    res = requests.get(httpserver.url_for("/"), params=query)
    res.raise_for_status()
```

now results in warnings:

```
mocking/test_httpserver_minimal.py::test_httpserver
  .../site-packages/pytest_httpserver/httpserver.py:216: DeprecationWarning: 'werkzeug.urls.url_decode' is deprecated and will be removed in Werkzeug 2.4. Use 'urllib.parse.parse_qs' instead.
    query = werkzeug.urls.url_decode(request_query_string)

mocking/test_httpserver_minimal.py::test_httpserver
  .../site-packages/werkzeug/urls.py:1195: DeprecationWarning: 'werkzeug.urls.url_unquote_plus' is deprecated and will be removed in Werkzeug 2.4. Use 'urllib.parse.unquote_plus' instead.
    url_unquote_plus(key, charset, errors),

mocking/test_httpserver_minimal.py::test_httpserver
  .../site-packages/werkzeug/urls.py:1196: DeprecationWarning: 'werkzeug.urls.url_unquote_plus' is deprecated and will be removed in Werkzeug 2.4. Use 'urllib.parse.unquote_plus' instead.
    url_unquote_plus(value, charset, errors),
```

coming from here:

https://github.com/csernazs/pytest-httpserver/blob/1df798a207f06772dc46a765d108a6edb0ab3f9b/pytest_httpserver/httpserver.py#L219

**Comment by csernazs** [comment]
Thanks for the issue! I think I will have time to fix it tomorrow.

Zsolt

**Comment by csernazs** [comment]
This has been published with version 1.0.7.
